lib/core/src/display/scene.rs

```diff
@@ -150,14 +145,22 @@ impl {
         let sub_logger      = logger.sub("symbols");
         let variables       = UniformScope::new(logger.sub("global_variables"),&context);
         let symbols         = SymbolRegistry::new(&variables,&stats,&context,sub_logger,on_change);
-        let shape           = Shape::default();
+        let window          = crate::system::web::window();
+        let inner_width     = window.inner_width().unwrap().as_f64().unwrap() as f32;
+        let inner_height    = window.inner_height().unwrap().as_f64().unwrap() as f32;
+        let shape           = Shape::new(inner_width, inner_height);
```

Comment:
This is shape initializer. It should be refactored to something like `Shape::from_window()` or something. This logic should not be in scene constructor.